env_data_folders/index.htm

```diff
@@ -83,6 +75,12 @@
         <a id="nameLink" class="navbar-brand" href="/" target="_self">%TITLE%</a>
       </div>
       <div class="collapse navbar-collapse" id="navbar-collapse-1">
+        <ul class="nav navbar-nav">
+          <li class="nav-main active"><a href="/">Main Controls</a></li>
+          <li class="nav-effects"><a href="/effects">Effects & Patterns</a></li>
```

Comment:
Those should be build dynamically based on the SectionFieldType?
Please correct.